website/docs/downloads/development-builds.md

```diff
@@ -90,10 +90,11 @@ function set_ci_status(workflow_file, os_name, description, page = 1) {
 }
 
 document.addEventListener("DOMContentLoaded", () => {
-    set_ci_status("linux.yml", "linux", "Linux");
-    set_ci_status("macos.yml", "macos", "macOS");
-    set_ci_status("windows-msys2.yml", "msys2", "Windows MSYS2 builds ¹");
-    set_ci_status("windows-msvc.yml", "windows", "Windows MSVC builds ²");
+    set_ci_status("linux.yml", "linux", "Linux x86-64");
+    set_ci_status("platforms.yml", "platforms", "Linux non-x86 ¹");
+    set_ci_status("macos.yml", "macos", "macOS x86-64 and ARM64");
```

Comment:
I think I'd prefer to just put a footnote on macOS and mention below the table that these are universal binaries for Intel and M1 (Mac users don't necessarily know what "ARM" is, but understand M1).